websites/C/CHub/presence.ts

```diff
@@ -0,0 +1,122 @@
+const presence = new Presence({
+		clientId: "1000715027994640404",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const { host } = document.location,
+		title = document.title.split("-");
+	let { title: dhname } = document,
+		{ pathname: path } = document.location;
+	path = path.replaceAll("/nighty", "");
+	if (title.length === 2) dhname = title[1].replace(/^[.\s]+|[.\s]+$/g, "");
+	let presenceData: PresenceData = {
+		largeImageKey: "chub",
```

Comment:
```suggestion
		largeImageKey: "https://i.imgur.com/Ut1y4n3.png",
```

Reply:
I suppose `largeImageKey` is an asset key of Discord Rich Presence, not an imgur link.

Reply:
We prefer imgur links